impl/proto/keytransparency_v1_service/keytransparency_v1_service.proto

```diff
@@ -70,3 +70,15 @@ service KeyTransparencyService {
     option (google.api.http) = { get: "/v1/domain/info" };
   }
 }
+
+// The KeyTransparencyAdminService API provides batch access to the directory of public keys. 
+service KeyTransparencyAdminService {
+  // BatchSetEntries uses an authorized_public key to perform a set request on multiple entries at once.  
```

Comment:
While these are longer than 80 chars, they match the length of the line below them, so I thought it would be easier on the eye to keep them on one line